h/templates/client/viewer.html

```diff
@@ -2,12 +2,11 @@
 
 (See gh2642 for rationale for 'ng-show="true"')
```

Comment:
It looks like at some point this comment has become detached from the part of the template that it's actually relevant to. The `See gh2642 for rationale for 'ng-show="true"'` refers to the `ng-show` property on the `<ul>` below, and not the `<selection-tabs>` component. Perhaps you could move this comment down so it's easier to understand what it's referring to?